apps/channels/tests/test_whatsapp_integration.py

```diff
@@ -193,10 +192,6 @@ def test_parse_messages(self, message, message_type):
             assert whatsapp_message.content_type == MESSAGE_TYPES.VOICE
             assert whatsapp_message.media_url == "http://example.com/media"
 
-    def test_unsupported_message_type_exception_raised(self):
-        with pytest.raises(UnsupportedMessageTypeException):
-            TwilioMessage.model_validate(json.loads(TwilioMessages.image_message()))
```

Comment:
What about updating one of the other tests should be updated to include unsupported message types to make sure that no exception is raised

Reply:
I think this was done in a later commit

Reply:
Yeah, apologies, I did not spend too much time on my commit history. Should have cleaned it up a bit